includes/admin/views/html-admin-page-addons.php

```diff
@@ -13,6 +13,8 @@
 
 ?>
 <div class="wrap woocommerce wc_addons_wrap">
+	<h1 class="screen-reader-text"><?php esc_html_e( 'Marketplace', 'woocommerce' ); ?></h1>
```

Comment:
Let's change this to `WooCommerce Marketplace` to match the title, this is only for screen readers but should match the main page title.

Reply:
Thanks, I updated the title now.